python/feathub/feature_views/transforms/join_transform.py

```diff
@@ -25,26 +25,26 @@ class JoinTransform(Transformation):
     def __init__(
         self,
         table_name: str,
-        feature_name: str,
+        expr: str,
     ):
         """
         :param table_name: The name of a Source or FeatureView table.
-        :param feature_name: The feature name.
+        :param expr: The feature expr.
```

Comment:
Does this support arbitrary FeatHub expression? If no, can you add doc explaining what is supported?